test/simplifyExpression/distributeSearch/distributeSearch.test.js

```diff
@@ -47,6 +47,16 @@ describe('distribute', function () {
       ['(-2x^2 * 3x - 2x^2 * -4)',
         '(-6x^3 + 8x^2)']
     ],
+    ['(x - 2)^2',
+      ['(x * (x - 2) - 2 * (x - 2))',
```

Comment:
agree that having `(x-2)(x-2)` would be good if we go to `(x * (x - 2) - 2 * (x - 2))`

however we could also use the rule based approach Ted's suggesting where we go straight to `x^2 -4x + 4` - I'd want to reference that special case rule though if we did that. And I tend to shy away from those approaches because it's more memorization based. Ideally we'd have `(x-2)(x-2)` and `(x * (x - 2) - 2 * (x - 2))` in substeps, imo at least